generics/helpers/utils.js

```diff
@@ -132,206 +133,249 @@ function assessmentRoles() {
+ * @returns {Array}  returns unique data in array.
+ */
 
 function removeDuplicatesFromArray(data, key) {
   let uniqueArray = [];
   let newMap = new Map();
- 
+
   uniqueArray = data.filter((singleData) => {
-  const element = newMap.get(singleData[key].toString());
-  if (!element) {
-    newMap.set(singleData[key].toString(), singleData);
-    return true;
-  } else {
+    const element = newMap.get(singleData[key].toString());
+    if (!element) {
+      newMap.set(singleData[key].toString(), singleData);
+      return true;
+    } else {
       return false;
     }
   });
- 
+
   return uniqueArray;
 }
 
 function convertStringToBoolean(stringData) {
-  let stringToBoolean = (stringData === "TRUE" || stringData === "true");
+  let stringToBoolean = stringData === "TRUE" || stringData === "true";
   return stringToBoolean;
 }
 /**
-  * check the uuid is valid
-  * @function
-  * @name checkIfValidUUID
-  * @returns {String} returns boolean.  
-*/
+ * check the uuid is valid
+ * @function
+ * @name checkIfValidUUID
+ * @returns {String} returns boolean.
+ */
 
 function checkIfValidUUID(value) {
-  const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
+  const regexExp =
+    /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
   return regexExp.test(value);
 }
 
 /**
-  * filter out location id and code
-  * @function
-  * @name filterLocationIdandCode
-  * @returns {Object} - Object contain locationid and location code array.  
-*/
+ * filter out location id and code
+ * @function
+ * @name filterLocationIdandCode
+ * @returns {Object} - Object contain locationid and location code array.
+ */
 
 function filterLocationIdandCode(dataArray) {
-    let locationIds = [];
-    let locationCodes = [];
-    dataArray.forEach(element=>{
-        if (this.checkIfValidUUID(element)) {
-            locationIds.push(element);
-        } else {
-            locationCodes.push(element);
-        }
-    });
-    return ({
-      ids : locationIds,
-      codes : locationCodes
-    });
+  let locationIds = [];
+  let locationCodes = [];
+  dataArray.forEach((element) => {
+    if (this.checkIfValidUUID(element)) {
+      locationIds.push(element);
+    } else {
+      locationCodes.push(element);
+    }
+  });
+  return {
+    ids: locationIds,
+    codes: locationCodes,
+  };
 }
 
 /**
-  * check whether string contains only number
-  * @function
-  * @name checkIfStringIsNumber
-  * @returns {Boolean} returns a Boolean value true/false
-*/
+ * check whether string contains only number
+ * @function
+ * @name checkIfStringIsNumber
+ * @returns {Boolean} returns a Boolean value true/false
+ */
 
 function checkIfStringIsNumber(str) {
   return /^[0-9]+$/.test(str);
 }
 
+/**
+ * generate Telemetry Event
+ * @function
+ * @name getTelemetryEvent
+ * @returns {Object} returns telemetry event object
+ */
+
+function getTelemetryEvent(message) {
```

Comment:
@ankitpws In utils telemetry function only what is common for all modules we should keep, rest all should be set and used as per the context.